tools/bump-version

```diff
@@ -11,16 +11,17 @@ program
   .requiredOption('--owner <owner>', 'owner of the current repository, usually "Integreat"')
   .requiredOption('--repo <repo>', 'the current repository, usually "integreat-webapp" or "integreat-react-native-app"')
   .requiredOption('--branch <branch>', 'the current branch')
+  .requiredOption('--release-notes <release-notes>', 'the release notes as stringified json')
   .option('--development-release', 'whether the release is a development release which is not delivered to production')
 
 const commitAndTag = async (versionName, versionCodeString, {
-  deliverinoPrivateKey, owner, repo, branch, developmentRelease
+  deliverinoPrivateKey, owner, repo, branch, developmentRelease, releaseNotes
 }) => {
   const appOctokit = await authenticate({ deliverinoPrivateKey, owner, repo })
   const versionFileContent = await appOctokit.repos.getContent({ owner, repo, path: VERSION_FILE, ref: branch })
 
-  const versionCode = parseInt(versionCodeString)
-  if (isNaN(versionCode)) {
+  const versionCode = versionCodeString ? parseInt(versionCodeString) : null
```

Comment:
Shouldn't we make versionCodeString required?

Reply:
See discussion here: https://github.com/Integreat/integreat-react-native-app/pull/286#discussion_r475562584